utility/start_server.sh

```diff
@@ -6,7 +6,7 @@ python /app/manage.py migrate
 if [ "${DJANGO_SETTINGS_MODULE}" = "config.settings.local" ] ; then
     echo "CREATING ADMIN USER FOR TESTING PURPOSES..."
     # Using key error as an indicator for whether or not to run database population and job scheduling scripts
-    echo "from django.contrib.auth import get_user_model; User = get_user_model(); User.objects.create_superuser('admin', 'admin@salesforce.com', 'password')" | python manage.py shell
+    echo "from django.contrib.auth import get_user_model; User = get_user_model(); User.objects.create_superuser('admin', 'admin@test.com', 'password')" | python manage.py shell
```

Comment:
I recommend using `example.com` here. `test.com` is an actual real-live domain owned by somebody, and we wouldn't want a bug to cause them to get unwanted email. 

`example.com`, on the other hand, is specifically designed to be used in documentation and code examples.

Reply:
GREAT catch jason said the same!